app/Console/UpdateEventStatusCommand.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Console;
+
+use CultuurNet\UDB3\Offer\OfferType;
+
+class UpdateEventStatusCommand extends AbstractUpdateOfferStatusCommand
+{
+    protected function configure(): void
+    {
+        $this->offerType = OfferType::EVENT();
+
+        $this
+            ->setName('event:status:update')
+            ->setDescription('Batch update status of events through SAPI 3 query.');
```

Comment:
Nit: You could also do this in the `AbstractUpdateOfferStatusCommand` if you want and make it concrete instead of abstract. For example something like this:
```
public function __construct(
    string $offerType,
    CommandBusInterface $commandBus,
    SearchServiceInterface $searchService
) {
    parent::__construct($commandBus);
	
    $this->setName($offerType . ':status:update');
    $this->setDescription(Batch update status of ' . $offerType . 's through SAPI 3 query.);
	
    $this->commandBus = $commandBus;
    $this->searchResultsGenerator = new ResultsGenerator(
        $searchService,
        ['created' => 'asc'],
        100
    );
}
```

The command base class of Symfony Console doesn't really do anything noteworthy before calling `configure()`. And calling `setName()` and `setDescription()` in `configure()` is basically the same as calling it in the constructor:
```
public function __construct($name = null)
    {
        $this->definition = new InputDefinition();

        if (null !== $name) {
            $this->setName($name);
        }

        $this->configure();

        if (!$this->name) {
            throw new LogicException(sprintf('The command defined in "%s" cannot have an empty name.', \get_class($this)));
        }
    }
```

I think they only do it like this (with `configure()`) so you wouldn't overwrite the constructor and potentially break the check that name should be required, while also preventing a thousand arguments in the constructor on the base class to configure it through that.

The reason I'm suggesting this is so you could get rid of the `abstract` if you want. But it's also okay like this for me since the child classes do very little aside from the setting the name and description.

Reply:
I like that. Will clean it up today and add the statusbar too.